packages/babel-generator/test/fixtures/types/ExportSpecifier4/output.js

```diff
@@ -1 +1 @@
-export * as foo, { bar } from "foo";
\ No newline at end of file
+export * as default from "foo";
```

Comment:
`ExportSpecifier4` is removed, then I renamed `ExportSpecifier16` into `ExportSpecifier4` to fill the hole.